packages/rich-text/src/plugins/List/index.tsx

```diff
@@ -163,6 +236,24 @@ const withCustomList = (options): WithOverride => {
     // Use our custom getListInsertFragment
     editor.insertFragment = getListInsertFragment(editor);
 
+    const { normalizeNode } = editor;
+    editor.normalizeNode = (entry) => {
+      const [node, path] = entry;
+
+      if (isList(node as CustomElement)) {
+        normalizeList(editor, path);
+        return;
```

Comment:
We should only return here if `normalizeList` did change the editor structure (by executing the code at line `196`). Otherwise, we will be skipping the call to `normalizeNode(entry)` at the bottom for no reason

When integrated with the other normalization code we have, it will result in some normalizers never being called.

Reply:
Fixed in f571378